client/my-sites/plans/index.js

```diff
@@ -31,6 +32,14 @@ export default function() {
 		);
 
 		page(
+			'/plans/my-plan/:site',
```

Comment:
I am not convinces, I like how it fits with `/plans/feature...` subtree. I like keeping consistent tree more than matching the plural/singular form